backend/api/parse_pdf.py

```
@@ -444,23 +635,67 @@ async def parse_pdf_with_progress(
+                file_contents_map[f"context_{i}"] = {
+                    "contents": b"",
+                    "filename": ctx_file.filename,
+                    "content_type": ctx_file.content_type,
+                    "error": str(e)
+                }
+        
         # Create semaphore for file processing to avoid overwhelming the system
         file_semaphore = asyncio.Semaphore(MAX_CONCURRENT_LLAMAPARSE)
         
-        async def process_file_with_semaphore(file_item, name):
+        async def process_file_contents_with_semaphore(file_data, name):
```

Ruff (0.13.3)

Unused function argument: `name`

(ARG001)